ndash/src/chunk/container_media_chunk_unittest.cc

```diff
@@ -228,20 +231,21 @@ TEST(ContainerMediaChunkTest, TestLoadFail) {
   constexpr int64_t kTestEndTime = 2;
   constexpr int32_t kTestChunkIndex = 3;
   const base::TimeDelta kSampleOffset = base::TimeDelta::FromMicroseconds(5000);
-  const std::unique_ptr<MediaFormat> media_format =
-      MediaFormat::CreateVideoFormat("1", "video/mp4", "h264", 2200000, 32768,
-                                     1234567, 640, 480, nullptr, 16, 45, 1.666);
+  std::unique_ptr<MediaFormat> media_format(MediaFormat::CreateVideoFormat(
+      "1", "video/mp4", "h264", 2200000, 32768, 1234567, 640, 480, nullptr, 16,
```

Comment:
q: are these two constants ("video/mp4" and "h264") present in chrome's media lib somewhere?

Reply:
No. There aren't.

Reply:
Ack.